index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Document</title>
+</head>
+<body>
+
+    <nav class = "navigation flex gap-xl align-center full-width">
+        <div class="nav-brand">Water-Commerce</div>
+        <div class = "input-wrapper flex grow-1 flex-center" >
+           <input class = "input-field" placeholder="Please Enter Text">
+       </div>
+       <div class = "flex gap-xl flex-center grow-1" >
+           <button class="btn btn-secondary">Login</button>
+           <i class="material-icons" >favorite_border</i>
+           <i class="material-icons" >shopping_cart</i>
+       </div>
+   </nav>
+
+   <div class = "m5-top"></div>
+
+   <section class = "preview container center-block flex space-between">
```

Comment:
no need of empty div, you can directly give margin to the section

Reply:
Understood. thanks